example/js/dialogs.js

```diff
@@ -96,6 +96,7 @@ ctrlrs.controller('errorDialogCtrl',['$scope','$uibModalInstance','$translate','
 	$scope.header = (angular.isDefined(data.header)) ? data.header : $translate.instant('DIALOGS_ERROR');
 	$scope.msg = (angular.isDefined(data.msg)) ? data.msg : $translate.instant('DIALOGS_ERROR_MSG');
 	$scope.icon = (angular.isDefined(data.fa) && angular.equals(data.fa,true)) ? 'fa fa-warning' : 'glyphicon glyphicon-warning-sign';
+	$scope.btnMessages = data.btnMessages || {};
```

Comment:
hmm actually do we need an `or` condition, as while we `setOpts`, we are already checking if it is empty.

Besides this, i think this PR looks good.